app/scripts/templates/tos.mustache

```diff
@@ -1,4 +1,4 @@
-<div id="main-content">
+<div id="main-content-legal">
```

Comment:
@shane-tomlinson: it would make sense, and I think this is the longer term change to be made here, but I'd rather merge this right now and clean up our class/id structure with #1661